components/support/rc_crypto/Cargo.toml

```diff
@@ -16,8 +16,7 @@ error-support = { path = "../error" }
 nss = { path = "nss" }
 libsqlite3-sys = { version = "0.15.0", features = ["bundled"] }
 hawk = { version = "3.0.0", default-features = false, optional = true }
-ece = { version = "1.0.1", optional = true }
+ece = { git = "https://github.com/eoger/rust-ece", branch = "dyn-crypto", default-features = false, features = ["serializable-keys"], optional = true }
```

Comment:
Note: this would have to be replaced by a published version of `rust-ece` (probably `1.1.0`).